### DISCUSSION TITLE
Does Renovate support CHANGELOGs for images from mono-repository?

### DISCUSSION THREAD

**Discussion by lictw** [renovatebot/renovate#28425] [Request Help]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

latest

### Please tell us more about your question or problem

If yes, how should I specify source directory where to find CHANGELOG.md? What should be a format of tags for compare?

**Comment by rarkins** [comment]
Which language /datasource?

**Reply by lictw** [reply]
Docker Images

**Reply by rarkins** [reply]
Please describe in more details

**Reply by lictw** [reply]
Can Renovate show changelogs as it normally does for Docker Images from CHANGELOG.md file in a root of provided as OCI label `org.opencontainers.image.source` repository, but for mono-repository, where from single repository more than one docker images does build? Each has own folder with CHANGELOG.md and so on.

**Reply by rarkins** [reply]
How would renovate know which subdirectory to look in?

**Reply by lictw** [reply]
It was the question) Seems that Renovate can do it in general, but only for node.js packages where it can obtain subdirectory, for docker we can provide it with some label also as url and ref.

**Reply by rarkins** [reply]
Renovate looks through releases for the version (e.g. `1.2.3` or `v1.2.3` or a combination of `packageName` and version, e.g. `foo-1.2.3`). For Docker the `packageName` may be quite long though if it includes the host, e.g. `docker.company.com/some/image`.
